python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -516,6 +519,17 @@ def run(self, sensorRef, templateIdList=None):
             mask  = subtractedExposure.getMaskedImage().getMask()
             mask &= ~(mask.getPlaneBitMask("DETECTED") | mask.getPlaneBitMask("DETECTED_NEGATIVE"))
 
+        # Compute ALZC correction kernel from matching kernel
+        # Here we use a constant kernel, just compute it for the center of the image.
+        # Investigating correction for spatially varying kernels is a subject of other tickets.
+        if self.config.doDecorrelation:
+            self.log.info("Running decorrelation correction.")
```

Comment:
Super picky, but "decorrelation correction" sounds like a double-negative to me. You are decorrelating, which corrects for over-correlating, right? Perhaps "running decorrelation" or "running image decorrelation"?